widget/edit.html

```diff
@@ -21,80 +21,175 @@ <h3 class="modal-title col-md-9" data-ng-bind="::header"></h3>
         </div>
       </div>
     </div>
-    <div class="form-group"
-      data-ng-class="{ 'has-error': editSankeyWidgetForm.resource.$invalid && editSankeyWidgetForm.resource.$touched }"
-      data-ng-if="modules">
-      <label for="resource" class="control-label">Resource<span class="text-danger">*</span></label>
-      <select name="resource" id="resource" class="form-control"
-        data-ng-options="module.type as module.name for module in modules" data-ng-model="config.resource" required
-        data-ng-change="fetchAttributes('target')">
-        <option value="">Select an Option</option>
-      </select>
-      <div data-cs-messages="editSankeyWidgetForm.resource"></div>
-    </div>
-    <div class="form-group" data-ng-if="config.resource">
-      <label for="resource" class="control-label">Filter Criteria</label>
-      <div data-cs-conditional data-mode="'queryFilters'" data-enable-nested-filter="enableNestedFilter"
-        data-fields="params.fields" data-ng-model="config.filters" data-form-name="'editSankeyWidgetForm'"
-        data-parent-form="editSankeyWidgetForm" data-reset-field="params.fields"></div>
-    </div>
-    <div class="form-group"
-      data-ng-class="{ 'has-error': editSankeyWidgetForm.sourceNodesField.$invalid && editSankeyWidgetForm.sourceNodesField.$touched }"
-      data-ng-if="config.resource && params.sourceNodeFields">
-      <label for="sourceNodesField" class="control-label">Source Nodes<span class="text-danger">*</span></label>
-      <select name="sourceNodesField" id="sourceNodesField" class="form-control"
-        data-ng-options="field.name as field.title for field in params.sourceNodeFields"
-        data-ng-model="config.sourceNodesField" required>
-        <option value="">Select an Field</option>
-      </select>
-      <div data-cs-messages="editSankeyWidgetForm.sourceNodesField"></div>
-    </div>
-    <div class="form-group"
-      data-ng-class="{ 'has-error': editSankeyWidgetForm.targetNodesField.$invalid && editSankeyWidgetForm.targetNodesField.$touched }"
-      data-ng-if="config.resource && params.targetNodePicklistFields">
-      <label for="targetNodesField" class="control-label">Target Nodes<span class="text-danger">*</span></label>
-      <select name="targetNodesField" id="targetNodesField" class="form-control"
-        data-ng-options="field.name as field.title for field in params.targetNodePicklistFields"
-        data-ng-model="config.targetNodesField" required>
-        <option value="">Select an Picklist</option>
-      </select>
-      <div data-cs-messages="editSankeyWidgetForm.targetNodesField"></div>
+
+    <label for="editSankeyWidgetForm" class="control-label">Data Source</label>
+    <span class="text-danger">*
+      <span
+        data-uib-tooltip="Select the 'Record Containing JSON Data' option if all the records to be rendered are in a single record of a certain field of the module.
```

Comment:
Localisation changes required for all static data.

Reply:
Localisation changes added